includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -1330,4 +1330,23 @@ public function create_setup_intent_ajax() {
 			);
 		}
 	}
+
+	/**
+	 * Add a url to the admin order page that links directly to the transactions detail view.
+	 *
+	 * @since 1.4.0
+	 *
+	 * @param WC_Order $order The context passed into this function when the user view the order details page in WordPress admin.
```

Comment:
👍 Let's add `@return` statement to the doc block as well.

Reply:
Updated with 1fd2228